src/main/java/com/epam/idea/core/model/Authority.java

```diff
@@ -16,9 +17,9 @@ public int getId() {
 		return id;
 	}
 
-	public static Authority getById(int id) {
+	public static Optional<Authority> getById(final int id) {
```

Comment:
Because this method might return null in case when the provided id is incorrect and I don't want to work with nulls so I'm using Optional as a safer alternative.